integrations/torchvision/tutorials/sparse-transfer-learning.md

```diff
@@ -0,0 +1,168 @@
+sparse_transfer_learning_modifiers:
+  - !ConstantPruningModifier
+    start_epoch: 0.0
+    params: __ALL_PRUNABLE__
+
+# Phase 2 Apply quantization
+sparse_quantized_transfer_learning_modifiers:
+  - !QuantizationModifier
+    start_epoch: eval(num_epochs - quantization_epochs)
+```
+
+The "Modifiers" encode how SparseML should modify the training process for Sparse Transfer Learning.
+- `ConstantPruningModifier` tells SparseML to pin weights at 0 over all epochs, maintaining the sparsity structure of the network
+- `QuantizationModifier` tells SparseML to quanitze the weights with quantization aware training over the last 6 epochs
+
+SparseML parses the instructions declared in the recipe and modifies the training loop accordingly before running the fine-tuning. 
+
+As a result, we end up with a 75% pruned and quantized ResNet-50 trained on Imagenette!
+
+#### Export to ONNX
+
+To deploy with DeepSparse, we need to export the model to ONNX.
+
+Run the following to do so:
+```bash
+sparseml.image_classification.export_onnx \
+  --arch_key resnet50 \
+  --checkpoint_path ./checkpoint.pth \
+  --dataset-path ./imagenette2-320
+```
+
+#### Modify a Recipe
+
+To transfer learn this sparsified model to other datasets you may have to adjust certain hyperparameters in this recipe and/or training script. Some considerations:
+
+- For more complex datasets, increase the number of epochs, adjusting the learning rate step accordingly
+- Adding more learning rate step milestones can lead to more jumps in accuracy
+- Increase the learning rate when increasing batch size
+- Increase the number of epochs if using SGD instead of the Adam optimizer
+- Update the base learning rate based on the number of steps needed to train your dataset
+
+To update a recipe, you can download the YAML file from SparseZoo, make updates to the YAML directly, and pass the local path to SparseML.
+
+Alternatively, you can use `--recipe_args` to modify a recipe on the fly. The following runs for 15 epochs instead of 10:
+
+```bash
+sparseml.image_classification.train \
+    --recipe zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe_type=transfer-classification \
+    --recipe_args '{"num_epochs": 15"}' \
+    --checkpoint-path zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none?recipe_type=transfer-classification \
+    --arch-key resnet50 \
+    --dataset-path ./imagenette2-320\
+    --batch-size 32
+```
+
+The `--recipe_args` are a json parsable dictionary of recipe variable names to values to overwrite the YAML-recipe.
+
+#### Dataset Format
+
+SparseML's Torchvision CLI conforms to the [Torchvision ImageFolder dataset format](https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html), where images are arranged into directories representing each class. To use a custom dataset with the SparseML CLI, you will need to arrange your data into this format.
```

Comment:
To use a custom dataset with the SparseML CLI, you will need to create your data in this format.

arrange doesn't sound like the right word here